public/app/plugins/datasource/grafana-azure-monitor-datasource/plugin.json

```diff
@@ -200,6 +200,66 @@
       },
       "headers": [{ "name": "x-ms-app", "content": "Grafana" }]
     },
+    {
+      "path": "loganalytics-resourcepickerdata",
+      "method": "POST",
+      "url": "https://management.azure.com",
+      "tokenAuth": {
```

Comment:
`tokenAuth` should be updated according to other routes

Reply:
Good catch - thanks!